### PR TITLE
Automatically update docs version

### PR DESCRIPTION
**Pull request by ulope**
This updates the bumpversion configuration to automatically update the documentation version along with the main app version.

Additionally:
- Convert README to restuctured text to allow proper rendering on PyPI
- Add README syntax check on travis
- Add version number on docs index page

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -70,9 +70,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'0.1.0'
+version = u'0.0.6'
```
Comment (czepluch) [inline docs/conf.py]: @LefterisJP suggested versioning it like 0.1.0. I don't remember the reasoning behind it, but I am fine with 0.0.6 too.

Comment (ulope) [reply docs/conf.py]: No that is just the current version from Konrad's deploy test. I think it's a good idea to bump to 0.1.0 for the preview.

**Comment by ulope** [comment]
Konrad approved verbally :)

File: docs/conf.py (hunk 1 above)
Comment (LefterisJP) [reply docs/conf.py]: yes 0.1.0 is the preview, not now

**Pull request merged**